paasta_tools/utils.py

```diff
@@ -323,9 +323,6 @@ class InstanceConfigDict(TypedDict, total=False):
     branch: str
     iam_role: str
     iam_role_provider: str
-    # the values for this dict can be anything since it's whatever
-    # spark accepts
-    spark_args: Dict[str, Any]
```

Comment:
we can probably move even more of the stuff in this dict out (e.g., aws_ebs_volumes) into more specific ones or just remove things entirely (cfs_period_us is probably unused), but that's for another day :p